ci-operator/config/redhat-operator-ecosystem/cvp/redhat-operator-ecosystem-cvp-ocp-4.7.yaml

```diff
@@ -12,9 +12,15 @@ resources:
       memory: 200Mi
 tests:
 - as: cvp-common-aws
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: dpp
```

Comment:
Not DPP, this would make you use our pools instead of yours.

Reply:
Changed to CVP